CONTRIBUTING.md

```diff
@@ -7,12 +7,12 @@ For bugs, feel free to open issues or contact us directly. Thank you for your su
 Even though we will gladly assist you in finishing up your PR, try to
 - keep all the crates compiling with *stable* rust (hide the eventual non-stable code under [`cfg`s](https://github.com/AFLplusplus/LibAFL/blob/main/libafl/build.rs#L26))
 - run `cargo +nightly fmt` on your code before pushing
-- check the output of `cargo clippy --all` or `./clippy.sh`
+- check the output of `cargo clippy --all` or `./scripts/clippy.sh`
```

Comment:
Doc fixes are always appreciated, feel free to open extra PRs for those as well :)